### PR TITLE
Add Google Gemini

### LINKED ISSUE #10449: Replace Google Bard with Gemini
### Brand Name

Gemini

### Official Resources for Icon and Color

There's [an SVG](https://www.gstatic.com/lamda/images/gemini_wordmark_landing_page_238102af073d0ae2763aa5.svg) on [their website](https://gemini.google.com/) - we should be able to get away with just the star. For the colour, maybe `#8ab4f8`?

### Additional Comments

Google Bard has [rebranded as Gemini](https://blog.google/products/gemini/bard-gemini-advanced-app/); we'll need to add Gemini and schedule Bard for removal in our next major release.

### PR DESCRIPTION
**Pull request by adamrusted**
image

**Issue:** closes #10449

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Using the shape from [this SVG](https://www.gstatic.com/lamda/images/discovery/gemini_dark_46398b861873dfeac5f00.svg), and the hex from the centre of the shape.

### CODE REVIEW

**Review by PeterShaggyNoble — APPROVED** [review]
SVG looks good 👍🏻

The colour's a tough call, though. The pink is the dominant colour in [this SVG](https://www.gstatic.com/lamda/images/bard_sparkle_v2_advanced.svg) and [this PNG](https://www.gstatic.com/lamda/images/advanced/gem-logo-inline-lrg_7d418af56850b209cccdb.png) - is that what we should go with, maybe? Or should it be blue, which is dominant in [their favicon](https://www.gstatic.com/lamda/images/gemini_favicon_b0ca051193f92ba9f912679dcf95b0efa9f201b6.svg).

Also, from that first SVG I linked, should we stick with the 2 stars for the icon, similar to Google Bard?

**Comment by PeterShaggyNoble** [comment]
Looking at that SVG I linked above, the filename would suggest that it's an updated icon for _Bard_ that we missed and _not_ for Gemini so I think we're fine with just the one star. If we disregard that SVG, though, then we'll need to look elsewhere for the colour - blue, as I mentioned above, is the dominant colour of their favicon and also, to me, their logo. We can either go with one of the blues from one of the `stop`s in the gradient, the calculated midpoint between the 2, the accent colour used for button links on the website, or, if in doubt, just resort to Google blue.

**Comment by adamrusted** [comment]
If we go with ye olde faithful [MeyerWeb Color Blender](https://meyerweb.com/eric/tools/color-blend/#4285F4:D96570:1:hex) we end up with `#8E75B2` as our midpoint. Happy to go with that?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5989,6 +5989,16 @@
             "hex": "7248B9",
             "source": "https://about.google/products/#all-products"
         },
+        {
+            "title": "Google Gemini",
+            "hex": "8679CC",
```
Comment (adamrusted) [inline _data/simple-icons.json:5994]:
```suggestion
            "hex": "8E75B2",
```

**Review by PeterShaggyNoble — APPROVED** [review]
Odd, I'd have expected something a bit more blue as the result 🤷🏻‍♂️

**Pull request merged**